libs/designsystem/src/lib/components/tabs/tabs.component.scss

```diff
@@ -21,11 +18,29 @@ ion-tabs {
     padding-right: $padding;
     width: 100%;
 
+    &.bottom-hidden {
+      display: none;
+    }
+
     @include media('>=large') {
       @include not-touch {
         margin: 0 auto;
         justify-content: flex-end;
       }
     }
   }
+
+  // move tabs to top of screen if not touch and large device
+  @include media('>=large') {
```

Comment:
Needs to be placed under:

```
    &.bottom-hidden {
      display: none;
    }
```

To work (at least in the unit test).

Reply:
Good point - media queries doesn't affect specificity, so yes, they have to be ordered after the rules they are overwriting 👍